src/styles/book.css

```diff
@@ -0,0 +1,218 @@
+body {
+  background-color: #efefef;
+}
+
+#main-container {
+  display: flex;
+  flex-direction: column;
+  padding: 15px 77px;
+}
+
+.display-books {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border: 1px solid #e8e8e8;
+  padding: 20px;
+  margin-bottom: 20px;
+  background-color: white;
+}
+
+#nav-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e8e8e8;
+  background-color: white;
+}
+
+#nav-container h1 {
+  color: #0290ff;
+  font-size: 1.875rem;
+  margin-left: 5rem;
+}
+
+#pages-container {
+  display: flex;
+  align-items: center;
+}
+
+#pages-container li {
+  list-style-type: none;
+  padding-left: 20px;
+  padding-right: 20px;
+  letter-spacing: 1px;
+}
+
+.user-icon {
+  border: 1px solid #e8e8e8;
+  padding: 13px;
+  border-radius: 24rem;
+  color: #0290ff;
+  margin-left: 77rem;
+  cursor: pointer;
+}
+
+.Books {
+  color: #121212;
+}
+
+.Categories {
+  color: #a3a3a3;
+}
+
+#all-btn {
+  margin-left: -5px;
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
+
+#all-btn button {
+  border: none;
+  background-color: white;
+  color: #4386bf;
+  font-size: 0.875rem;
+  cursor: pointer;
+}
+
+.line-1 {
+  height: 1.5rem;
+  border: solid 1px #f5f6fa;
+}
+
+.category {
+  width: 3.063rem;
+  height: 0.125rem;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.author {
+  width: 6.688rem;
+  height: 2.188rem;
+  font-size: 0.875rem;
+  color: #4386bf;
+}
+
+#form-container {
+  padding: 60px 77px;
+}
+
+.percentage-sign h3 {
+  font-size: 2rem;
+  font-weight: normal;
+  margin-bottom: 0;
+}
+
+#form-container h3 {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-size: 1.25rem;
+  font-weight: bold;
+  color: #888;
+}
+
+.book-title {
+  width: 55.5rem;
+  height: 1.23rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 13px 40px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  font-size: 1rem;
+}
+
+.input-author {
+  height: 1.23rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 13px 40px;
+  width: 25rem;
+  font-size: 1rem;
+}
+
+.submit-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  color: white;
+  cursor: pointer;
+  font-size: 1rem;
+}
```

Comment:
- [ ] [OPTIONAL] I would suggest you to fit all input fields and button in one line to match the given design perfectly.

|Your design| Given design|
|---|-----|
|<img width="1124" alt="Screenshot 2023-05-20 at 7 52 17 PM" src="https://github.com/Shaheryar0054/bookstore-using-react/assets/44798044/ff670a19-5a41-43e6-bb7d-88f2924dd72b">|<img width="1124" alt="Screenshot 2023-05-20 at 7 59 34 PM" src="https://github.com/Shaheryar0054/bookstore-using-react/assets/44798044/d43eb2bf-7bb4-4158-a3ca-adcbbff3d32a">|